### PR TITLE
try to save byte by move string to variable

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/utils/isWeb.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
-export default typeof window !== 'undefined' &&
-  typeof window.HTMLElement !== 'undefined' &&
-  typeof document !== 'undefined';
+const UNDEFINED = 'undefined';
```
Comment (NMinhNguyen) [inline src/utils/isWeb.ts:1]: Would you save more bytes by using `let`? 😆

```suggestion
let UNDEFINED = 'undefined';
```

Comment (NMinhNguyen) [inline src/utils/isWeb.ts:1]: Would you save more bytes by using `let`? 😆

```suggestion
let UNDEFINED = 'undefined';
```

Comment (bluebill1049) [reply src/utils/isWeb.ts:1]: I have been thinking about it change all const to let

`1:5  error  'UNDEFINED' is never reassigned. Use 'const' instead  prefer-const`

**Comment by bluebill1049** [comment]
Failed :)

**Pull request closed without merging**